test/html/test.ml

```diff
@@ -289,6 +289,9 @@ let source_files_post408 =
 
 let source_files_pre410 = [ ("bugs_pre_410.ml", [ "Bugs_pre_410/index.html" ]) ]
 
+let source_files_stop_dead_link_doc =
```

Comment:
The convention would be to call this `source_files_post404`. This way more tests can be added to this list.